### ISSUE TITLE
On iOS foreground push notification works only after fresh installation.

### ISSUE THREAD

**Issue by samrakchan** [dpa99c/cordova-plugin-firebasex#824]
On iOS push notifications do not work while the app is in the foreground when the app is existing app is updated. However, it starts working on the fresh installation of the app.

While the app is in the background it works on both occasions on fresh installation and as well as on the version update of the app.

**Comment by dpa99c** [comment]
You have not filled out the issue template so have provided insufficient information therefore this issue will be closed as invalid.

Please follow the [documented steps](https://github.com/dpa99c/cordova-plugin-firebasex?tab=readme-ov-file#reporting-a-bug-or-problem) for reporting an issue.

If you are able to reproduce the issue in the example project, please open a new issue and fill out the issue template in full, including steps to reproduce in the example project and details of test devices used to reproduce the issue.
